### DISCUSSION TITLE
String getting truncated

### DISCUSSION THREAD

**Discussion by sheikzaidh** [fastapi/fastapi#6729] [Questions]
from client the client side the entire string getting pass while coming to the backend the string getting truncated

**Comment by krishnardt** [comment]
Please follow the issue template with proper example, so that someone can understand issue and help you.

**Comment by tiangolo** [comment]
Thanks @krishnardt! :coffee: 

Sorry, I can't help you if you don't follow the issue template and provide more information @sheikzaidh .

If you still have problems please create a new issue and follow the template step by step.

Given that, I'm gonna close this one.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
